pkgs/cronet_http/lib/cronet_client.dart

```diff
@@ -170,13 +171,19 @@ class CronetClient extends BaseClient {
 
   @override
   void close() {
-    if (_ownedEngine) {
+    if (!_isClosed && _ownedEngine) {
       _engine?.close();
     }
+    _isClosed = true;
   }
 
   @override
   Future<StreamedResponse> send(BaseRequest request) async {
+    if (_isClosed) {
```

Comment:
Do we need to worry about the `CronetEngine` that never gets a `.close()` call in that narrow case where `.close` is called synchronously following `send`?

Reply:
No, the _CronetEngine should eventually get GCed.